runtime/common/dispatch.rs

```diff
@@ -174,6 +189,7 @@ where
 				Self::Fungible(h) => h.call(handle),
 				Self::Nonfungible(h) => h.call(handle),
 				Self::Refungible(h) => h.call(handle),
+				Self::NativeFungible(f) => todo!(),
```

Comment:
У тебя не вызывается реализация CommonEvmHandler, компилятор мог её выкинуть.
```suggestion
				Self::NativeFungible(h) => h.call(handle),
```